.ci/scripts/pulp_file-tests.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/bash -e
```

Comment:
This is replacing the original shebang.
I believe the usual way to accomplish is to add `set -e` below.
Also consider using `set -eu`.